apis/python/src/tiledbsoma/io/ingest.py

```diff
@@ -2650,13 +2670,17 @@ def to_h5ad(
     measurement_name: str,
     *,
     X_layer_name: Optional[str] = "data",
-    obs_id_name: str = "obs_id",
-    var_id_name: str = "var_id",
+    obs_id_name: Optional[str] = None,
+    var_id_name: Optional[str] = None,
     obsm_varm_width_hints: Optional[Dict[str, Dict[str, int]]] = None,
     uns_keys: Optional[Sequence[str]] = None,
 ) -> None:
     """Converts the experiment group to `AnnData <https://anndata.readthedocs.io/>`_
-    format and writes it to the specified ``.h5ad`` file. Arguments are as in ``to_anndata``.
+    format and writes it to the specified ``.h5ad`` file.
+
+    Arguments are as in ``to_anndata``.
+
+    TO DO: doc more params
```

Comment:
Can we we create an issue/story and remove this from user-facing docs?

Reply:
https://github.com/single-cell-data/TileDB-SOMA/issues/2071

https://github.com/single-cell-data/TileDB-SOMA/pull/2028/commits/896cbc0d4118436d2e89ec2d9e57dc13a1c4258a